server/validate.js

```diff
@@ -46,7 +44,69 @@ exports.validateCreateConversation = [
   check("recipientId", "recipientId is required").not().isEmpty(),
   (req, res, next) => {
     const errors = validationResult(req);
+    if (!errors.isEmpty())
+      return res.status(400).json({ errors: errors.array() });
+    next();
+  },
+];
+
+exports.validateProfile = [
```

Comment:
Great work adding validation middleware to the routes 💯

Reply:
Thank you :)